<?xml version="1.1"?>
<launch>
<arg name="port" default="/dev/ttyUSB0"/>
<node name="get_sensor_data_server" pkg="temphum" type="getTempAndHumidity.py" output="screen"> 
<param name="port" value="$(arg port)" />   
</node>
</launch>
